core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -368,6 +368,11 @@ public MetricKey build() {
           .setDescription("An estimate of the blocks heap size")
           .setMetricType(MetricType.GAUGE)
           .build();
+  public static final MetricKey MASTER_RPC_PENDING_JOBS =
+      new Builder("Master.RpcPendingJobs")
```

Comment:
How about `Master.RpcQueueLength`? is it accurate?

Reply:
RpcCallQueueLength？

Reply:
`Master.RpcQueueLength` is better. RpcCallQueueLength is from hdfs, and there is call in the context of hdfs, for Alluxio, just call it `RpcQueueLength`.

Reply:
ok👌